### PR TITLE
GRPC V2 : Get Baker List

### PR DESCRIPTION
**Pull request by tds-1**
## Purpose

Added support for GRPC V2 `GetBakerList`

## Issue Link 

https://github.com/Concordium/concordium-java-sdk/issues/192

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 - Fixed incorrect transaction event with respect to a transaction status for when a delegator decreased its stake.
 - Added support for GRPC V2 `GetBlockInfo` for retrieving information of the block being queried.
 - Added support for GRPC V2 `GetTokenomicsInfo` for acquiring various tokenomics related information for a block.
+- Added support for GRPC V2 `GetBakerList`
```
Comment (EmilLa1) [inline CHANGELOG.md:17]:
```suggestion
- Added support for GRPC V2 `GetBakerList` for retrieving the bakers at a certain block.
```

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetBakerList.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetBakerList", mixinStandardHelpOptions = true)
+public class GetBakerList implements Callable<Integer> {
```
Comment (EmilLa1) [inline concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetBakerList.java:18]: Please provide one sentence of what the example does.

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetBakerListTest.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.delegatesTo;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientV2GetBakerListTest {
```
Comment (EmilLa1) [inline concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetBakerListTest.java:25]: Please add one sentence of what the intention is with the test. I.e. testing the mapping functionality.

File: CHANGELOG.md (hunk 1 above)
Comment (parv0888) [reply CHANGELOG.md:17]: done

File: concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetBakerList.java (hunk 2 above)
Comment (parv0888) [reply concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetBakerList.java:18]: done

File: concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetBakerListTest.java (hunk 3 above)
Comment (parv0888) [reply concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetBakerListTest.java:25]: done

**Review by EmilLa1 — APPROVED** [review]
Looks good

**Pull request merged**